.cloud-build/execute_changed_notebooks_helper.py

```diff
@@ -336,12 +379,79 @@ def get_changed_notebooks(
 
     return notebooks
 
+def _save_results(results: List[NotebookExecutionResult],
+                  artifacts_bucket: str,
+                  results_file: str):
+
+    artifacts_bucket = artifacts_bucket.replace("gs://", "").split('/')[0]
+
+    # read in existing prior results data
+    print("Reading existing accumulative results ...")
+    rows = []
```

Comment:
Is there a reason why this can't be simplified to saving the list of `NotebookExecutionResult`'s into a JSON file?